ui/src/buckets/actions/thunks.ts

```diff
@@ -101,10 +124,10 @@ export const createBucket = (bucket: OwnBucket) => async (
       throw new Error(resp.data.message)
     }
 
-    const newBucket = normalize<Bucket, BucketEntities, string>(
-      resp.data,
-      bucketSchema
-    )
+    const b: BucketWithLabel = resp.data
+    b.labels = []
```

Comment:
I think this should be taken care of by normalization?

Reply:
If I get rid of this part then it crashes when trying to create a new bucket. (Same for deleting, etc.) Is it possible I need to make a change to the normalization or something else?

Reply:
I imagine that this is crashing because the api is expecting the Buckets to resemble the GenBuckets type, when in reality they've been merged to include specific properties like labels and readableRetention. If we want to change the type to GenBuckets we should probably strip those properties in the function, since I imagine those are still being passed in